docs/docs/apps/README.md

```diff
@@ -28,6 +28,7 @@ The vast majority of integrated apps on Pipedream are free to use in your workfl
 - [Salesforce (REST API)](https://pipedream.com/apps/salesforce_rest_api)
 - [SendinBlue](https://pipedream.com/apps/sendinblue)
 - [ServiceNow](https://pipedream.com/apps/servicenow)
+- [Snowflake](https://pipedream.com/apps/snowflake)
```

Comment:
```suggestion
- [Snowflake](https://pipedream.com/apps/snowflake)
```

Reply:
Changed 'Snowflake' to 'snowflake' to follow sentence case rule in Google's Developer docs style guide